src/plugins/apm_oss/server/tutorial/instructions/apm_agent_instructions.ts

```diff
@@ -60,19 +61,27 @@ var apm = require('elastic-apm-node').start({curlyOpen}
   // ${i18n.translate(
     'apmOss.tutorial.nodeClient.configure.commands.useIfApmRequiresTokenComment',
     {
-      defaultMessage: 'Use if APM Server requires a token',
+      defaultMessage: 'Use if APM Server requires a secret token',
     }
   )}
   secretToken: '${secretToken}',
 
   // ${i18n.translate(
     'apmOss.tutorial.nodeClient.configure.commands.setCustomApmServerUrlComment',
     {
-      defaultMessage: 'Set custom APM Server URL (default: {defaultApmServerUrl})',
+      defaultMessage: 'Set the custom APM Server URL (default: {defaultApmServerUrl})',
       values: { defaultApmServerUrl: 'http://localhost:8200' },
     }
   )}
-  serverUrl: '${apmServerUrl}'
+  serverUrl: '${apmServerUrl}',
+
+  // ${i18n.translate(
+    'apmOss.tutorial.nodeClient.configure.commands.setCustomServiceEnvironmentComment',
+    {
+      defaultMessage: 'Set the service environment (default: production)',
```

Comment:
```suggestion
      defaultMessage: 'Set the service environment',
```

We're not changing the defaults in the agents or through APM server yet. So this is just adding the environment for discoverability and hopefully users want to use _something_ here.